src/Streamly/Internal/Data/Stream/IsStream/Reduce.hs

```diff
@@ -983,7 +983,7 @@ splitOnSuffixSeqAny _subseq _f _m = undefined
 chunksOf
     :: (IsStream t, Monad m)
     => Int -> Fold m a b -> t m a -> t m b
-chunksOf n f = fromStreamD . D.chunksOf n f . toStreamD
+chunksOf n f = fromStreamD . D.groupsOf n f . toStreamD
```

Comment:
No need to change this. This is a deprecated module. Why make a breaking change in this?

Reply:
There is no breaking change in this.